tests/behaviour/typeql/mod.rs

```diff
@@ -19,4 +19,12 @@
  * under the License.
  */
 
+// mod get;
+mod define;
+mod delete;
+mod insert;
+mod match_;
+mod rule_validation;
 mod steps;
+// mod undefine;
```

Comment:
if we don't have anything for `get` and `undefine` let's remove them? 

Although I believe there's a `get` package, how did we decide which modules are included here or not? Actually, do we need this file at all?

cc @dmitrii-ubskii

Reply:
We need it, but there are some steps that are implemented in the next PRs.